libs/payments/capability/src/lib/capability.manager.spec.ts

```diff
@@ -17,36 +17,36 @@ import { CapabilityManager } from './capability.manager';
 
 describe('CapabilityManager', () => {
   describe('getClients', () => {
-    let manager: CapabilityManager;
-    let mockContentfulManager: ContentfulManager;
-    let mockResult: ServicesWithCapabilitiesResultUtil;
+    let capabilityManager: CapabilityManager;
+    let contentfulManager: ContentfulManager;
+    let servicesResult: ServicesWithCapabilitiesResultUtil;
 
     beforeEach(async () => {
-      mockResult = {} as ServicesWithCapabilitiesResultUtil;
-      mockContentfulManager = {
+      servicesResult = {} as ServicesWithCapabilitiesResultUtil;
+      contentfulManager = {
```

Comment:
Seeing as these are assigned to both an empty object and to a jest.fn, these definitely deserve the `mock` prefix